### PR TITLE
update more info log in upgrade script

### PR DESCRIPTION
**Pull request by Daniel-Fan**

### CODE REVIEW

File: common/scripts/upgrade_to_continuous_delivery.sh [hunk 1]
```diff
@@ -180,6 +184,7 @@ function switch_to_continous_delivery() {
     done < <(oc get sub --all-namespaces | grep ibm-common-service-operator | awk '{print $1" "$2}')
     
     success "Updated all ibm-common-service-operator subscriptions successfully."
+    info "Please wait for a moment to let ibm-common-service-operator upgrade all foundational services."
```
Comment (horis233) [inline common/scripts/upgrade_to_continuous_delivery.sh:187]: Just an advice. Does the following message look better?
```
Please wait a moment for ibm-common-service-operator to upgrade all foundational services
```

Comment (Daniel-Fan) [reply common/scripts/upgrade_to_continuous_delivery.sh]: It looks good! Updated.

Comment (horis233) [reply common/scripts/upgrade_to_continuous_delivery.sh]: Thanks!

**Review by horis233 — APPROVED** [review]
/lgtm

**Pull request merged**